session/pessimistic_test.go

```diff
@@ -391,3 +393,30 @@ func (s *testPessimisticSuite) TestOptimisticConflicts(c *C) {
 	tk.MustExec("commit")
 	tk.MustQuery("select * from conflict").Check(testkit.Rows("1 3"))
 }
+
+func (s *testPessimisticSuite) TestWaitLockKill(c *C) {
```

Comment:
maybe we need a complex test: update 2 locked rows in different regions?

Reply:
I've changed the code following @coocood 's advise, now `actionPessimisticLock.handleRegion` would not reset the `killed` flag. 

> update 2 locked rows in different regions

What's the case you would like to cover? and how is it differs from one region?